device-types/Arista/DCS-7280CR3K-32P4A.yaml

```diff
@@ -0,0 +1,96 @@
+---
+manufacturer: Arista
+model: DCS-7280CR3K-32P4A
+slug: arista-dcs-7280cr3k-32p4a
+part_number: DCS-7280CR3K-32P4A
+u_height: 1
+is_full_depth: true
```

Comment:
Please add the `airflow` definition if you have or can find it. This is often available in the products datasheet, but sometimes it is not. If you cannot find it just let us know and resolve this comment.